src/common/options/rgw.yaml.in

```diff
@@ -1148,6 +1148,22 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_nfs_frontends
```

Comment:
can you please add a `see_also` for `rgw_frontends`? the docs in https://docs.ceph.com/en/latest/radosgw/config-ref/ will add links

Reply:
sure